packages/flutter/lib/src/widgets/media_query.dart

```diff
@@ -128,11 +128,11 @@ class MediaQueryData {
   /// notifications so that you can update your [MediaQueryData] when the
   /// window's metrics change. For example, see
   /// [WidgetsBindingObserver.didChangeMetrics] or [Window.onMetricsChanged].
-  MediaQueryData.fromWindow(ui.Window window)
+  MediaQueryData.fromWindow(ui.Window window, { ui.Brightness platformBrightness })
```

Comment:
@goderbauer thoughts on this API change? It seems likely to be non-breaking. I could instead create a mocked window ... but this seemed more straightforward

Reply:
Instead of adding the extra slightly awkward one-off API, could you just do the following in `app.dart`?

```dart
MediaQueryData data = MediaQueryData.fromWindow(WidgetsBinding.instance.window);
if (!kReleaseMode) {
  data = data.copyWith(platformBrightness: debugBrightnessOverride);
}
```

Reply:
Ahh, I missed the existing `copyWith` method - that seems more straightforward